charts/radarr/Chart.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v2
+name: radarr
+description:
+  Radarr is a movie collection manager for Usenet and BitTorrent users. It can
+  monitor multiple RSS feeds for new movies and will interface with clients and
+  indexers to grab, sort, and rename them.
+type: application
+version: 0.1.0
+appVersion: 5.20.2
+keywords:
+  - radarr
```

Comment:
Maybe I would add `Movie`, `Movies` or the like, neat picking here.

Reply:
We're gonna go with `films` for our UK friends